pennylane/plugins/tests/pytest.ini

```diff
@@ -1,4 +0,0 @@
-[pytest]

```

Comment:
I have no idea why this file shows up here. It is deleted locally...and even says so here.

Reply:
It is also not in the files of this branch here on github. Probably safe to merge.

Reply:
Oh! I added this file a couple of PRs back! I didn't realise it was being deleted

Reply:
This file needs to stay, as it 'registers' @antalszava's `skip_unsupported` mark. Without it being registered, pytest raises a warning with each use.